moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua

```diff
@@ -101,6 +104,22 @@ function get_account_name(account_id)
     return account_id
 end
 
+function get_identity_name()
```

Comment:
I think the indenting might be off in here?

Reply:
yeah it is, fixed.